tcwebhooks-web-ui/src/main/resources/buildServerResources/WebHook/js/editWebhook.js

```diff
@@ -536,22 +536,25 @@ function populateWebHookDialog(id){
 			$j('#webHookFormContents select#payloadFormatHolder').val(webhook.payloadTemplate + "_" + webhook.payloadFormat).change();
 			
 			$j('#buildTypeSubProjects').prop('checked', webhook.subProjectsEnabled);
-			$j.each(webhook.builds, function(){
-				var thing = $j(this.buildTypeName).text();
-				console.log(thing);
-				 if (this.enabled){
-			 	 	$j('#buildList').append('<p style="border-bottom:solid 1px #cccccc; margin:0; padding:0.5em;"><label><input checked onclick="updateSelectedBuildTypes();" type=checkbox style="padding-right: 1em;" name="buildTypeId" value="' + this.buildTypeId + '"class="buildType_single">' + htmlEscape(this.buildTypeName) + '</label></p>');
-				 } else {
-				 	 $j('#buildList').append('<p style="border-bottom:solid 1px #cccccc; margin:0; padding:0.5em;"><label><input onclick="updateSelectedBuildTypes();" type=checkbox style="padding-right: 1em;" name="buildTypeId" value="' + this.buildTypeId + '"class="buildType_single">' + htmlEscape(this.buildTypeName) + '</label></p>');
-				 }
+			$j.each(webhook.builds, function() {
+				console.log('Adding subproject', this.buildTypeId, 'named', this.buildTypeName);
+				var isChecked = '';
+				if (this.enabled) {
+					isChecked = ' checked';
+				}
+				var label = $j('<label><input' + isChecked + ' onclick="updateSelectedBuildTypes();" type=checkbox style="padding-right: 1em;" name="buildTypeId" value="' + this.buildTypeId + '"class="buildType_single"></label>');
```

Comment:
this change has dropped the checkbox

Reply:
What's the error? Does it just not show up at all? If you place a breakpoint there in the browser, do you see anything out of the ordinary? As far as I can tell the code should work..

Reply:
There is no error, it just only shows the label, but with no checkbox inside it. 

I agree, the code looks legit.